lib/generators/shopify_app/add_webhook/add_webhook_generator.rb

```diff
@@ -3,12 +3,20 @@
 module ShopifyApp
   module Generators
     class AddWebhookGenerator < Rails::Generators::Base
+      include WebhookTopicValidator
+
       source_root File.expand_path('../templates', __FILE__)
 
       class_option :topic, type: :string, aliases: "-t", required: true
       class_option :address, type: :string, aliases: "-a", required: true
 
       def init_webhook_config
+        begin
+          is_valid_topic?(topic)
+        rescue ShopifyApp::WebhookTopicValidator::InvalidTopic => e
+          invalid_topic_shell_message
+          raise
+        end
```

Comment:
rather than using begin you could just use implicit rescue inside this whole function. Then it would look like this:

```ruby
def init_webhook_config
  is_valid_topic?(topic)
  ...
rescue ShopifyApp::WebhookTopicValidator::InvalidTopic => e
  ...
end
```